docs/modules/ROOT/pages/configuration/global-filters.adoc

```diff
@@ -0,0 +1,95 @@
+= Configuring Global Filters on KubeArchive
+
+KubeArchive allows cluster owners to configure global filters. Those are applied
+to any KubeArchive namespace containing a `KubeArchiveConfig` custom resource.
+
+To configure the global filters create a `KubeArchiveConfig` in the
+"kubearchive" namespace. Because it is created on the "kubearchive" namespace
+these filters are considered global:
+
+[source,yaml]
+----
+---
+apiVersion: kubearchive.kubearchive.org/v1alpha1
+kind: KubeArchiveConfig
+metadata:
+  name: kubearchive
+  namespace: kubearchive
+spec:
+  resources:
+    - selector:
+        apiVersion: v1
+        kind: Pod
+      archiveOnDelete: "true"
+----
+
+== Interaction With Local Filters
+
+Global filters only work in namespaces containing a `KubeArchiveConfig` custom
+resource. These can be understood as "local" filters in opposition with global
+filters. Both global and local filters are combined and used for that specific
+namespace.
```

Comment:
you mention that both the local and global filters are combined, but I think we should explicitly mention that we combine the results using an or, not an and